nixos/modules/services/misc/radarr.nix

```diff
@@ -11,6 +11,13 @@ in
     services.radarr = {
       enable = mkEnableOption "Radarr";
 
+      package = mkOption {
+        description = "Radarr package to use";
+        default = pkgs.radarr;
```

Comment:
```suggestion
        default = pkgs.radarr;
        defaultText = literalExpression "pkgs.radarr
```

Reply:
applied in https://github.com/NixOS/nixpkgs/pull/168141/commits/0c0be749646ce642bf50acfee75a484996556c0e